### PR TITLE
Adding new card for OpenShift Level Program to the OpenShift Learning resources page

### PR DESCRIPTION
**Pull request by kevinhatchoua**

### CODE REVIEW

File: docs/quickstarts/openshift-level-up-program/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kind: QuickStarts # kind must always be "QuickStarts"
+name: openshift-level-up-program
+tags: # If you want to use more granular filtering add tags to the quickstart
+  - kind: bundle
+    value: OpenShift
```
Comment (Hyperkid123) [inline docs/quickstarts/openshift-level-up-program/metadata.yaml:5]: This should be lowercase: `openshift`

File: docs/quickstarts/openshift-level-up-program/openshift-level-up-program.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: openshift-level-up-program
+  externalDocumentation: true
+spec:
+  version: 0.1
+  type:
+    text: Documentation
```
Comment (dayleparker) [inline docs/quickstarts/openshift-level-up-program/openshift-level-up-program.yaml:9]: Is this the correct content type?

**Comment by kevinhatchoua** [comment]
Following conversations with stakeholders, we will not merge this into the console. Thanks!

**Comment by kevinhatchoua** [comment]
Closing PR

**Pull request closed without merging**